Hw_05/hw05.Rmd

```diff
@@ -118,12 +188,29 @@ str_no_scent <- function(x){
   	)
   )
 }
+
+Scented_all$CodeReview <- str_no_scent(Scented_all$Review)
+Scented_all$CodedReview_f <- factor(Scented_all$CodeReview, labels = c("scent", "noscent"))
+print(Scented_all)
 ```
 
 ```{r}
-
+Scented2_Plot <- Scented_all %>%
```

Comment:
You can leave small notes on your code so when you go back to them later on, you will know what you did.